taf/tests/test_updater/test_update/test_update_valid.py

```diff
@@ -730,9 +731,13 @@ def test_update_with_invalid_last_validated_commit(origin_auth_repo, client_dir)
     # Step 1: Clone the repositories
     clone_repositories(origin_auth_repo, client_dir)
 
-    # Step 2: Set the last validated commit to an invalid commit (e.g., non-existent SHA)
-    invalid_commit_sha = "0000000000000000000000000000000000000000"
-    origin_auth_repo.set_last_validated_commit(invalid_commit_sha)
+    # Step 2: Delete the last validated commit file to simulate an invalid commit (non-existent SHA)
```

Comment:
The update should be successful if there is not last validated commit. We expect it to start from th beginning. We should definitely keep this test, but also need to test the case when last validated commit is set to an invalid commit. The updater might fail. If that is the case, just comment out that test and I'll work on that later